src/login/logind-action.c

```diff
@@ -299,12 +300,49 @@ static int handle_action_sleep_execute(
         return handle_action_execute(m, handle, ignore_inhibited, is_edge);
 }
 
+static int manager_handle_action_secure_attention_key(
+                Manager *m,
+                bool is_edge,
+                char *seat) {
+        int r;
+        Seat *o;
+        _cleanup_free_ char *p = NULL;
+
+        assert(m);
+        assert(seat);
+
+        if (!is_edge)
+                return 0;
+
+        o = hashmap_get(m->seats, seat);
+        if (!o)
+                return 0;
+
+        p = seat_bus_path(o);
+        if (!p)
+                return -ENOMEM;
```

Comment:
so in systemd functions either should actively log about all their possible errors, or not do log at all, and leave the logging to the caller (LOG_DEBUG level debug logging is always ok, and doesn#t count as logging in this context).

We follow this rule pretty much always these days, even if older code in our codebase (which logind is) is not that great at following this rule yet.

Given you add this funciton anew, make sure to follow this rule, i.e. figure out if you function shall log about errors (in which case make sure to log about them all), or not (in which case consider adding debug logging, but not more).

This one should really be of the "logging" kind. Hence, please change this `return -ENOMEM;` line here to `return log_oom();`.